partiql-eval/src/test/kotlin/org/partiql/eval/internal/PartiQLEvaluatorTest.kt

```diff
@@ -1320,7 +1321,9 @@ class PartiQLEvaluatorTest {
         )
 
         internal fun assert() {
-            val statement = parser.parse(input).root
+            val parseResult = parser.parse(input)
+            assertEquals(1, parseResult.statements.size)
+            val statement = parseResult.statements[0]
```

Comment:
Here is an example where the _parse_ and _parseAll_ would introduce functionality without introducing complexity for the consumer.